File: PRAK205-2210817220019-DHEA_APRILINDA_UTAMI.py
tinggi, miring = (int(input()) for _ in range(2))
alas = (miring**2-tinggi**2)**0.5
print("Alas =",round(alas),"cm")
print("Tinggi =",round(tinggi),"cm")
print("Keliling =",round(tinggi + miring + alas),"cm")
print("luas =",round(tinggi*alas/2),"cm^2")

File: PRAK205-2210817220019-DHEA_APRILINDA_UTAMI.c
#include <stdio.h>
#include <math.h>

int main()
{
    int  tinggi, miring, alas;
    scanf("%d", &tinggi);
    scanf("%d", &miring);
    alas = sqrt(miring*miring-tinggi*tinggi);
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n",tinggi);
    printf("Keliling = %d cm\n", tinggi+miring+alas);
    printf("luas = %d cm^2\n", tinggi*alas/2);

}